docs/3.0/build.md

```diff
@@ -0,0 +1,247 @@
+
+```json
+{
+  ...
+  "scripts": {
+    ...
+    "build": "..."
+  }
+  ...
+}
+```
+
+Supported build script hooks are:
+
+  - `"build"`: executed for all platform builds
+  - `"build:android"`: executed for Android builds
+  - `"build:ios"`: executed for iOS builds
+
+#### Example: TypeScript
+
+When generating a "Compiled" Tabris.js app using the `tabris init` command it is already configured to use the TypeScript compiler. The details are explained [here](./typescript.md#setup).
+
+#### Example: Babel.js
+
+Babel.js may be used to add support for specific [JavaScript features not natively supported by Tabris.js](./runtime.md).
+
+To do so, install the Babel transpiler and the necessary plug-ins. The `--save-dev` option will add the dependencies to your `package.json`:
+
+```
+npm install --save-dev babel-cli babel-plugin-transform-es2015-modules-commonjs
+```
+
+Create a `.babelrc` file in the root of your project:
+
+```json
+{
+  "plugins": ["transform-es2015-modules-commonjs"]
+}
+```
+
+Include the following build script in the `scripts` sections of your `package.json`:
+
+```json
+{
+  "scripts": {
+    "build": "babel --compact false --out-dir dist/ src/"
+  }
+  ...
+}
+```
+
+Let the `main` field point to the *compiled* version of `app.js` in `dist/`:
+
+```json
+{
+  "main": "dist/app.js",
+  ...
+}
+```
+
+To test the setup run `npm run build` or simply `tabris serve`.
+
+### The config.xml file
+
+The minimal build configuration you need is a `cordova/config.xml` file that describes your app. It contains information like the id of your app, its version, icons and splash screens. The format of the `config.xml` is the same as a standard [Cordova config.xml](https://cordova.apache.org/docs/en/4.0.0/config_ref_index.md.html#The%20config.xml%20File) file. A minimal example config could look like this:
+
+```xml
+<?xml version='1.0' encoding='utf-8'?>
+<widget id="my.first.app" version="1.0.0">
+  <name>HelloWorld</name>
+  <description>
+    A sample Tabris.js application.
+  </description>
+  <author email="dev@tabrisjs.com" href="https://tabrisjs.com">
+    Tabris.js Team
+  </author>
+</widget>
+```
+
+#### Integrating Cordova plugins
+
+To add a set of Apache Cordova plug-ins you only need to add them to `config.xml` using the `<plugin />` tag. It allows you to add plug-ins using an ID, an HTTP URL or a git URL.
+
+For example, to add the [Cordova Camera Plugin](http://plugins.cordova.io/#/package/org.apache.cordova.camera), you'd add this line:
+
+```xml
+<plugin name="cordova-plugin-camera" spec="^2.3.0" />
+```
+
+You can integrate all available [Cordova Plugins](http://plugins.cordova.io/#/) by including them in your `config.xml`.
+
+**Important:** You can install all available Cordova Plugins. Most of the Plugins will work out of the box. However, since Tabris.js uses a **native UI** and **no HTML5**, plugins that rely on an HTML5 UI (i.e. the DOM) won't work.
+
```

Comment:
We had that, it was a link to a github issue label. I removed it because no one ever used it. We had that discussion 3 years ago and my opinion hasn't changed: If we want to have a list we need to maintain it ourselves.

Reply:
You can integrate ~all~ available

Agree that a repository of such needs to be kept.  The Github issue solution of past was a bit clunky; you had to wade through several pages, and then click on something of interest to determine what its outcome was.  If it's a single page, it would make user contributions (editing through the GitHub UI) pretty easy.